cmd/snappy/cmd_update.go

```diff
@@ -53,7 +53,7 @@ const (
 	shutdownTimeout = "+10"
 )
 
-var shutdownMsg = i18n.G("snappy autopilot triggered a reboot to boot into an up to date system -- temprorarily disable the reboot by running 'sudo shutdown -c'")
```

Comment:
Note I'd much rather change this to “snappy autoupdate needs to reboot to finish an update. Cancel with 'shutdown -c'” which (I think) communicates the same thing, but keeps it under 80 cols. Wall doesn't word-wrap, so the longer it is the more chances of it being untidy.